### PR TITLE
Fix Drupal coding standards errors/warnings

### PR DESCRIPTION
**Pull request by fafnirical**
```
FILE: gesso/gesso.theme
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 1 | ERROR | [x] The PHP open tag must be followed by exactly one
   |       |     blank line
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------


FILE: gesso/includes/form.inc
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 1 | ERROR | [x] The PHP open tag must be followed by exactly one
   |       |     blank line
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------


FILE: gesso/includes/gesso.drush.inc
----------------------------------------------------------------------
FOUND 15 ERRORS AND 1 WARNING AFFECTING 13 LINES
----------------------------------------------------------------------
   1 | ERROR   | [x] The PHP open tag must be followed by exactly one
     |         |     blank line
  11 | ERROR   | [x] Short array syntax must be used to define arrays
  11 | ERROR   | [x] The first index in a multi-value array must be
     |         |     on a new line
  11 | ERROR   | [x] Short array syntax must be used to define arrays
  13 | ERROR   | [x] Short array syntax must be used to define arrays
  16 | ERROR   | [x] Short array syntax must be used to define arrays
  20 | ERROR   | [x] Short array syntax must be used to define arrays
  24 | ERROR   | [x] Closing parenthesis of array declaration must be
     |         |     on a new line
  41 | WARNING | [ ] Unused variable $description.
  73 | ERROR   | [x] Short array syntax must be used to define arrays
  74 | ERROR   | [x] Comments may not appear after statements
  75 | ERROR   | [x] Expected one space after the comma, 3 found
  75 | ERROR   | [x] Comments may not appear after statements
 122 | ERROR   | [x] Short array syntax must be used to define arrays
 148 | ERROR   | [x] Short array syntax must be used to define arrays
 167 | ERROR   | [x] Short array syntax must be used to define arrays
----------------------------------------------------------------------
PHPCBF CAN FIX THE 15 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------


FILE: gesso/includes/html.inc
----------------------------------------------------------------------
FOUND 4 ERRORS AND 1 WARNING AFFECTING 5 LINES
----------------------------------------------------------------------
  1 | ERROR   | [x] The PHP open tag must be followed by exactly one
    |         |     blank line
 13 | ERROR   | [x] Short array syntax must be used to define arrays
 15 | ERROR   | [x] Short array syntax must be used to define arrays
 20 | WARNING | [x] A comma should follow the last multiline array
    |         |     item. Found: 'x_ua_compatible'
 40 | ERROR   | [x] Closing brace must be on a line by itself
----------------------------------------------------------------------
PHPCBF CAN FIX THE 5 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------


FILE: gesso/README.md
----------------------------------------------------------------------
FOUND 0 ERRORS AND 8 WARNINGS AFFECTING 8 LINES
----------------------------------------------------------------------
  11 | WARNING | Line exceeds 80 characters; contains 121 characters
  20 | WARNING | Line exceeds 80 characters; contains 82 characters
  27 | WARNING | Line exceeds 80 characters; contains 93 characters
  36 | WARNING | Line exceeds 80 characters; contains 83 characters
  50 | WARNING | Line exceeds 80 characters; contains 102 characters
  99 | WARNING | Line exceeds 80 characters; contains 82 characters
 102 | WARNING | Line exceeds 80 characters; contains 101 characters
 104 | WARNING | Line exceeds 80 characters; contains 128 characters
----------------------------------------------------------------------


FILE: gesso/sass/README.md
----------------------------------------------------------------------
FOUND 0 ERRORS AND 11 WARNINGS AFFECTING 11 LINES
----------------------------------------------------------------------
  3 | WARNING | Line exceeds 80 characters; contains 459 characters
  7 | WARNING | Line exceeds 80 characters; contains 217 characters
  9 | WARNING | Line exceeds 80 characters; contains 109 characters
 12 | WARNING | Line exceeds 80 characters; contains 136 characters
 15 | WARNING | Line exceeds 80 characters; contains 84 characters
 18 | WARNING | Line exceeds 80 characters; contains 170 characters
 27 | WARNING | Line exceeds 80 characters; contains 103 characters
 33 | WARNING | Line exceeds 80 characters; contains 138 characters
 37 | WARNING | Line exceeds 80 characters; contains 332 characters
 39 | WARNING | Line exceeds 80 characters; contains 230 characters
 41 | WARNING | Line exceeds 80 characters; contains 354 characters
----------------------------------------------------------------------
```

### CODE REVIEW

**Comment by coreylafferty** [comment]
@dcmouyard any issue with me accepting these?

File: includes/gesso.drush.inc [hunk 1]
```diff
@@ -38,7 +41,6 @@ function gesso_drush_help($section) {
  * Implements drush_hook_COMMAND_validate().
  */
 function drush_gesso_validate($name = NULL) {
-  $description = drush_get_option('description');
```
Comment (dcmouyard) [inline includes/gesso.drush.inc:41]: Don’t we still need this line since we still have a description option?

**Review by dcmouyard — CHANGES_REQUESTED** [review]

File: includes/gesso.drush.inc (hunk 1 above)
Comment (fafnirical) [reply includes/gesso.drush.inc:41]: The variable `$description` isn't used in this function

Comment (dcmouyard) [reply includes/gesso.drush.inc:41]: Ah, then it’s definitely okay to remove that line.

**Review by dcmouyard — APPROVED** [review]

**Pull request merged**